WalletWasabi/Helpers/NBitcoinHelpers.cs

```diff
@@ -13,17 +13,6 @@ namespace WalletWasabi.Helpers
 {
 	public static class NBitcoinHelpers
 	{
-		public static string HashOutpoints(IEnumerable<OutPoint> outPoints)
```

Comment:
`/// <remarks><see cref="IDisposable"/> is implemented. Do not forget to use `using` or dispose any instance of this class.</remarks>`

This remark can be removed now too since we have CA2000 diagnostic turned on.

Reply:
Done